content/docs/concepts/components/stack.md

```diff
@@ -92,7 +92,57 @@ To create a custom stack, simply create customized build and run images containi
 > `io.buildpacks.stacks.bionic` as its identifier so long as it will work with buildpacks that declare compatibility with the 
 > `io.buildpacks.stacks.bionic` stack.
 
-### Resources
+## Mixins
+
+Mixins provide a way to document OS-level dependencies that a stack provides to buildpacks. Mixins can be provided at build-time
```

Comment:
This comment is late (sorry), but I find “OS-level dependencies” a bit confusing. “OS-level” can imply a feature of the kernel.  And “dependencies” usually refers to something that is *required* for operation, whereas these mixing document things that are *provided* by the stack.  Might “features” work better?

And it would be helpful to document that
  1. mixins do not cause packages to be installed (this caused me some confusion when I first learned about buildpacks)
  2. there are no explicit semantics associated with to mixin strings, brings the `run:` and `build:` prefixes
  3. any restrictions that there might be in the string. Can they include spaces or colons?

Reply:
Thanks for the feedback @briandealwis! We'll definitely incorporate some of these into the docs. We would also take a PR if you're interested in contributing.